ui/dashboards/src/components/Dashboard/Dashboard.stories.tsx

```diff
@@ -223,7 +223,9 @@ function formatProviderParameters(dashboardState: DashboardResource) {
     },
     withTimeRange: {
       props: {
```

Comment:
Bad typing caught by the addition of the parameter typing. Really glad I decided to figure that out while I was in here.